### DISCUSSION TITLE
Enabling three text inputs to be next to each other

### DISCUSSION THREAD

**Discussion by MTPGthb** [mProjectsCode/obsidian-meta-bind-plugin#275] [Q&A]
Hello.

Last week, I was able to have three Text inputs next to eachother within the same line. When I loaded in the same file with the same vault it changed the spacing. Was there a change in the update? Is there any possible way to change the spacing? I've tried changing most within Minimal and Style settings.

thank you.

**Comment by mProjectsCode** [comment]
> Was there a change in the update?

There was no update in the last 4 weeks except if you updated via BRAT.

It would help to know how the relevant part of your note looks.
